x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -20,26 +32,28 @@ import { AlertIndex } from './view/alerts';
 import { HostList } from './view/hosts';
 import { PolicyList } from './view/policy';
 import { PolicyDetails } from './view/policy';
-import { HeaderNavigation } from './components/header_nav';
-import { setupIngestManager } from '../../../../ingest_manager/common';
 import { EuiThemeProvider } from '../../../../../legacy/common/eui_styled_components';
-
-export async function setupDeps(coreStart: CoreStart) {
-  // TODO handle a 500 from the ingest manager
-  await setupIngestManager(coreStart.http);
-}
+import { IngestManagerSetup } from '../../../../ingest_manager/public';
+import { HeaderNavigation } from './components/header_nav';
 
 /**
  * This module will be loaded asynchronously to reduce the bundle size of your plugin's main bundle.
  */
 export function renderApp(
   coreStart: CoreStart,
   depsStart: EndpointPluginStartDependencies,
+  ingestManager: IngestManagerSetup,
```

Comment:
I have a comment further below that will possibly avoid having to pass this and can probably get it from `depsStart`.